iOS/ViewControllers/Courses/CourseViewController.swift

```diff
@@ -4,6 +4,7 @@
 //
 
 // swiftlint:disable file_length
+// swiftlint:disable type_body_length
```

Comment:
You can disable multiple SwiftLint rules in a single line

```suggestion
// swiftlint:disable file_length type_body_length
```